server/src/main/scala/org/http4s/server/middleware/authentication/DigestAuth.scala

```diff
@@ -62,115 +110,179 @@ object DigestAuth {
     *                       purposes anymore).
     * @param nonceBits The number of random bits a nonce should consist of.
     */
-  def apply[F[_]: Sync, A](
+  def applyF[F[_]: Concurrent: Timer, A](
```

Comment:
Do you want this to be `applyF` or `apply` in the long run?  I don't generally like overloads, but I think they're compatible.

Reply:
They're compatible, save for the default arguments. I'd expect this needs to revert back to `apply`, once the non-`F[_]`'d `apply` is deleted, but that'll be a breaking change (possibly with a deprecation cycle, duplicating the signature that just passes through to the underlying `apply`)

Reply:
I was just thinking, if you want to call it `apply` now, you probably could.  Although they both take default arguments, so, actually, I think you can't.  Suggestion withdrawn.